ovirt/resource_ovirt_vm.go

```diff
@@ -457,9 +451,110 @@ func resourceOvirtVMCreate(d *schema.ResourceData, meta interface{}) error {
 func resourceOvirtVMUpdate(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*ovirtsdk4.Connection)
 	vmService := conn.SystemService().VmsService().VmService(d.Id())
-	paramVM := ovirtsdk4.NewVmBuilder()
+	vmBuilder := ovirtsdk4.NewVmBuilder()
 	attributeUpdated := false
 
+	// Block that update VM Basic parameters:
+	// Name, Memory, Cluster, CPU params
+	if name, ok := d.GetOk("name"); ok {
+		vmBuilder.Name(name.(string))
+	}
+
+	if memory, ok := d.GetOk("memory"); ok {
+		// memory is specified in MB
+		vmBuilder.Memory(int64(memory.(int)) * int64(math.Pow(2, 20)))
+	}
+
+	cluster, err := ovirtsdk4.NewClusterBuilder().
+		Id(d.Get("cluster_id").(string)).Build()
+	if err != nil {
+		return err
+	}
+	vmBuilder.Cluster(cluster)
+
+	if ha, ok := d.GetOkExists("high_availability"); ok {
+		highAvailability, err := ovirtsdk4.NewHighAvailabilityBuilder().
+			Enabled(ha.(bool)).Build()
+
+		if err != nil {
+			return err
+		}
+		vmBuilder.HighAvailability(highAvailability)
+	}
+
+	cpuTopo := ovirtsdk4.NewCpuTopologyBuilder().
+		Cores(int64(d.Get("cores").(int))).
+		Threads(int64(d.Get("threads").(int))).
+		Sockets(int64(d.Get("sockets").(int))).
+		MustBuild()
+
+	cpu, err := ovirtsdk4.NewCpuBuilder().
+		Topology(cpuTopo).
+		Build()
+	if err != nil {
+		return err
+	}
+	vmBuilder.Cpu(cpu)
+
+	//paramVM.Initialization(initialization)
+	if v, ok := d.GetOk("initialization"); ok {
+		initialization, err := expandOvirtVMInitialization(v.([]interface{}))
+		if err != nil {
+			return err
+		}
+		if initialization != nil {
+			vmBuilder.Initialization(initialization)
+		}
+	}
+
+	var err_updateresult error
+	_, err_updateresult = vmService.Update().Vm(vmBuilder.MustBuild()).Send()
```

Comment:
please change err_updateresult to err. I can imagine you wanted a distinct the various error variables, 
but go vet agains that, and suggests simply 'err'. I want to keep the coding convention here.

Reply:
Agree with you.
Updated error name https://github.com/oVirt/terraform-provider-ovirt/pull/198/commits/99fa6edfaf5a210efd36c4d1da87e161cdf54285